Plugins/Statistics/GitStatistics/LineCounter.cs

```diff
@@ -21,7 +21,7 @@ public sealed class LineCounter
 
         public void FindAndAnalyzeCodeFiles(string filePattern, string directoriesToIgnore, IEnumerable<string> filesToCheck)
         {
-            var extensions = filePattern.Replace("*", "").Split(';').ToHashSet(StringComparer.InvariantCultureIgnoreCase);
+            var extensions = LinqExtensions.ToHashSet(filePattern.Replace("*", "").Split(';'), StringComparer.InvariantCultureIgnoreCase);
```

Comment:
❔ Intentional change? Do we not want to just remove `LinqExtensions` now that `System.Linq.Enumerable` has this method?

Reply:
This is outside the scope of this PR. But will certainly raise a follow up issue.